### PR TITLE
Teleportation and controller selection are now webxr features features

### PR DESCRIPTION
**Pull request by RaananW**
Closing #7290 

Teleportation and selection are now features that are a part of the default xr experience. to use it outside of the default experience (with typescript types):

```javascript
const input = /* init babylon's xrInput class */
const pointerSelection = <WebXRControllerPointerSelection>featuresManager.enableFeature(WebXRControllerPointerSelection.Name, "latest", {
    xrInput: input
});

const teleportation = <WebXRMotionControllerTeleportation>featuresManager.enableFeature(WebXRMotionControllerTeleportation.Name, "latest", {
    floorMeshes: floorMeshes,
    xrInput: input
});
// connect both so that the selection will not be shown when teleportation is shown
result.teleportation.setSelectionFeature(result.pointerSelection);
```

To enable or disable, use the attach and detach functions:

```javascript
const teleportation = <WebXRMotionControllerTeleportation>featuresManager.enableFeature(WebXRMotionControllerTeleportation.Name, "latest", {
    floorMeshes: floorMeshes,
    xrInput: input
}, false /* optionally do not auto-attach */);

// enabling it:
teleportation.detach();

//disabling it:
teleportation.attach()
```

### CODE REVIEW

**Comment by deltakosh** [comment]
Fantastic! can you fix the conflict and update the webxr doc? Thanks a ton!

**Comment by RaananW** [comment]
I love resolving conflicts that I personally created :-)

I am planning on re-writing the doc at the end of this month, with everything done so far.
